src/test/src/controllers/soleTraderCorrespondenceAddressManualController.test.ts

```diff
@@ -133,9 +133,9 @@ describe("POST" + SOLE_TRADER_MANUAL_CORRESPONDENCE_ADDRESS, () => {
 
 // Test for no addressCountry, will return 302.
 describe("POST" + SOLE_TRADER_MANUAL_CORRESPONDENCE_ADDRESS, () => {
-    xit("should return status 302", async () => {
+    it("should return status 302", async () => {
```

Comment:
Test is returning 400, description should reflect this, could also add an expect for the error message to be shown